### ISSUE TITLE
Possibility of Alkane Hydrocarbons past Hexane

### ISSUE THREAD

**Issue by drummerdude2003** [al132mc/alchemistry#179]
The possibility to string together hydrocarbons, specifically in the form of alkanes, is already possible up to Hexane, with 6 carbon moles being needed. In reality, these can be strung near infinitely, being used in the creation of things like polystyrene polymers. of course, that would require thousands of these being strung together, so that isn't in the scope of this mod
One thing, however, is "Higher alkanes", which are alkanes that utilize 9 or more carbon atoms, which I believe could be utilized in creating a super fuel. The one drawback to this is the fact that we can only go up to 'Hentriacontane', which would past this point would require more than a stack of hydrogen.

TLDR: I want to make more hydrocarbons past Hexane, to make fuels better than coal, or most existing forms of fuel in the game

**Comment by drummerdude2003** [comment]
(here's the wiki for higher alkanes)

[https://en.wikipedia.org/wiki/Higher_alkanes](url)

**Comment by Dark-Arcana** [comment]
@Timbroglio is this something you would like to implement?

**Comment by Timbroglio** [comment]
Well, Chemlib doesn't actually add fuel functionality to these materials. I think it's an interesting idea, but also that chemical-based fuels would fit better in with the planned Techemistry rewrite
